cms/static/js/views/course_video_settings.js

```diff
@@ -416,7 +416,7 @@ function($, Backbone, _, gettext, moment, HtmlUtils, StringUtils, TranscriptSett
                 preferred_languages: self.selectedLanguages,
                 global: false   // Do not trigger global AJAX error handler
             }, function(data) {
-                if (data.transcript_preferences) {
```

Comment:
Was it working correctly before ???? I think we don't use the data, is that why ?

Reply:
data is the parameter which contains response, I changed this to support no provider selection case.